app/recordtransfer/managers.py

```diff
@@ -53,8 +53,8 @@ def export_csv(
 class UploadSessionManager(models.Manager):
     """Custom manager for UploadSession model."""
 
-    def get_expired(self) -> query.QuerySet:
-        """Return all upload sessions that have expired."""
+    def get_expirable(self) -> query.QuerySet:
+        """Return all expired upload sessions that can be set to EXPIRED."""
```

Comment:
This function has some overlap with `get_deletable`. This one should *only* return upload sessions that can be expired. The way it works right now, it can return some that are expirable, and some that are deletable.

I've made a suggestion on the doc string for how this should work:

```suggestion
    def get_expirable(self) -> query.QuerySet:
        """Return all expired upload sessions that can be set to EXPIRED.

        A session can be expired if it has not been interacted with in at least
        UPLOAD_SESSION_EXPIRE_AFTER_INACTIVE_MINUTES minutes. Additionally, the session must be
        linked to an existing InProgressSubmission.

        If a session is *not* linked to an InProgressSubmission, and matches the same last
        interaction criteria above, then it will be returned by get_deletable instead.
        """
```

Reply:
Addressed by: https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/431/commits/1a375334156ecf11c05f45b76004183d4f64a2fb